testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -5071,6 +5071,75 @@ QUnit.module('the \'fieldTemplate\' option', moduleSetup, () => {
     });
 });
 
+
+QUnit.module('the \'valueChangeEvent\' option', moduleSetup, () => {
```

Comment:
To simplify search when u copy test name from the browser page
```suggestion
QUnit.module('the "valueChangeEvent" option', moduleSetup, () => {
```

Reply:
https://github.com/DevExpress/DevExtreme/pull/22772/commits/42f158af2eaaba57402677771dc9b9eb3bac416d